easy_my_coop_connector/tests/test_payment.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from unittest.mock import Mock, patch
+
+import requests
+
+from odoo.fields import Date
+
+from odoo.addons.easy_my_coop.tests.test_base import EMCBaseCase
+
+from .test_data import AP_CREATE_RESULT, SR_VALIDATE_RESULT, dict_to_dump
+
+
+class EMCPaymentConnectorCase(EMCBaseCase):
+    def setUp(self):
+        super().setUp()
+        self.backend = self.browse_ref("easy_my_coop_connector.emc_backend_demo")
+        self.env["emc.binding.account.journal"].create(
+            {
+                "backend_id": self.backend.id,
+                "internal_id": self.bank_journal.id,
+                "external_id": 1,
+            }
+        )
+
+    def test_post_payment_sends_and_binds_request(self):
+        srequest = self.browse_ref("easy_my_coop.subscription_request_1_demo")
+        with patch.object(requests, "post") as mock_get:
```

Comment:
do you know about [`requests_mock`](https://pypi.org/project/requests-mock/)?

Reply:
no, I'll take a look next time :-)